readthedocs/templates/projects/project_advanced.html

```diff
@@ -13,7 +14,7 @@
 
 {% block project_edit_content %}
   <form method="post" action=".">{% csrf_token %}
-    {{ form.as_p }}
+    {% crispy form %}
     <p>
       <input style="display: inline;" type="submit" value="{% trans "Save" %}">
```

Comment:
@stsewd 
`{% crispy form %}` renders
```html
<form>
  <!-- csrf token -->
  <!-- content -->
</form>
```
And we have wrapped this in outer `<form>` tag and it is finally rendering something like this.
![Screenshot from 2019-03-27 21-50-05](https://user-images.githubusercontent.com/29149191/55093556-99816f00-50da-11e9-9288-61077b84a01e.png)

and so the `Save` button, which is rendered outside `<form>` tag, is not working anymore. We need to fix this before merging this in `rel` branch

docs: https://django-crispy-forms.readthedocs.io/en/latest/crispy_tag_forms.html#fundamentals

Reply:
https://github.com/rtfd/readthedocs.org/pull/5544